src/types.ts

```diff
@@ -10,15 +10,22 @@ type BaseTask = {
   description: string;
 };
 
+interface Container extends HTMLElement {
+  _reactRootContainer?: any;
+}
+
 export type RunStaticTaskArgs = {
   getElement: () => React.ReactElement;
   container: HTMLElement;
 };
-
+export type RunStaticTaskArgsWithReactRoot = {
+  getElement: () => React.ReactElement;
+  container: Container;
+};
 export type StaticTask = BaseTask & {
   scale?: string;
   type: 'static';
-  run: (args: RunStaticTaskArgs) => Promise<string>;
+  run: (args: RunStaticTaskArgs | RunStaticTaskArgsWithReactRoot) => Promise<string>;
```

Comment:
Why do we need to change this type?

Reply:
I think you could do this check within your task rather than adding the union

Reply:
After that I think we are good to go

Reply:
Hey, sorry for the delay! fixed